tools/release/stage-release.ts

```diff
@@ -136,9 +136,9 @@ class StageReleaseTask extends BaseReleaseTask {
     // Note: When updating the commit messages here. Please also update the
     // release publish script to detect the new commit messages.
     if (needsVersionBump) {
-      this.git.createNewCommit(`chore: bump version to ${newVersionName} w/ changelog`);
+      this.git.createNewCommit(`release: bump version to ${newVersionName} w/ changelog`);
```

Comment:
We also need to update the verification of commit message in the publish script: https://github.com/angular/components/blob/33a02a7d9c7ae9fe0185d66a9e72a551aaad2b75/tools/release/publish-release.ts#L147